.goreleaser.yml

```diff
@@ -99,3 +99,17 @@ snapcrafts:
         # things they allow:
         # https://snapcraft.io/docs/reference/interfaces.
         plugs: ["home", "network"]
+
+scoop:
+  bucket:
+    owner: newrelic
+    name: newrelic-cli
+  commit_author:
+    name: nr-developer-toolkit
+    email: developer-toolkit-team@newrelic.com
```

Comment:
We don't want folks emailing the team directly, use: `opensource@newrelic.com`

Reply:
Used a secret instead. Should be obfuscated in CI now 😎